git_wrapper/clone.py

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env python
+"""This module acts as an interface for git repository management tasks"""
+
+import logging
+import os
+
+import git
+from future.utils import raise_from
+
+from git_wrapper.base import GitWrapperBase
+from git_wrapper import exceptions
+
+
+logger = logging.getLogger(__name__)
```

Comment:
I agree here, we should be injecting the logger into each class instead of setting up a new one each time. I think some rotation is in order for the library that will makes this a little easier.